src/idlcxx/src/backendCpp11Utils.c

```diff
@@ -324,7 +324,7 @@ get_cpp11_templ_type_const_value(const idl_constval_t *constval)
     return NULL;
   assert(constval->value.str);
   len = strlen(constval->value.str);
-  if (!(str = malloc(len + 2 /* quotes */ + 1)))
+  if ((str = malloc(len + 2 /* quotes */ + 1)) == 0)
```

Comment:
Which warning does this fix? Better use `NULL` instead of `0`, it's C code rather than C++.

Reply:
These changes were made to fix MSVC warning 4706 (assignment within conditional expression: https://docs.microsoft.com/en-us/cpp/error-messages/compiler-warnings/compiler-warning-level-4-c4706?view=msvc-160).
By having the explicit comparison operator, this warning does not occur, while the assignment still works as expected.

Using NULL in stead of 0 is a good catch though.